style.css

```diff
@@ -0,0 +1,76 @@
+.header__language-ru {
+     font-family: Montserrat, sans-serif;
+     font-style: normal;
+     font-weight: 500;
+     font-size: 20px;
+     line-height: 24px;
+  }
+
+.new-collection__title {
+     font-family: Montserrat, sans-serif;
+     font-style: normal;
+     font-weight: 500;
+     font-size: 30px;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Locc/pullRequest?prid=5906701)